Source/Lib/ASM_SSSE3/EbMcp_Intrinsic_AVX512.c

```diff
@@ -0,0 +1,457 @@
+#include "EbMcp_SSSE3.h"
```

Comment:
missing license header

Reply:
the AVX512 code should be moved to the AVX2 project

Reply:
Fixed